src/dmd/dscope.d

```diff
@@ -69,6 +69,8 @@ enum SCOPE
     // The following are mutually exclusive
     printf        = 0x4_0000, /// printf-style function
     scanf         = 0x8_0000, /// scanf-style function
+
+    doublepop     = 0x10_0000, /// for "scope-less" with statements
```

Comment:
I beg your pardon?

Reply:
would you like another name?

Reply:
It has no relation with the change at all.